schemas/dto/jobExecutionUserInfoCollection.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Collection of job executions unique user info",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "jobExecutionUserInfo": {
```

Comment:
Hi. I recommend to use plural form. You can replace with "jobExecutionsUserInfo" or  "jobExecutionUsersInfo".